cg/server/dto/samples/collaborator_samples_request.py

```diff
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+
+
+class CollaboratorSamplesRequest(BaseModel):
+    enquiry: str
+    customer: str
+    limit: int = 50
```

Comment:
Why do we have a limit here? I would guess that all samples matching the query are relevant.

Reply:
I mean, if the rest of the parameters result in 60 samples, on what grounds can we hide 10 of them?

Reply:
It was there so I kept it. I guess we don't want an infinite list in the drop down and that the search the customers do usually narrows it down to fewer than 50 entries anyway

Reply:
I think it is fundamentally wrong to limit this query. As you say, the query sent by the customer should already limit the query enough, and if not, there is no good reason to hide some of them. I do not know if this is needed for when the customer starts typing but I hope that there is a better way. Either way, since it was not introduced here, you can consider it out of scope.